src/store/store.ts

```diff
@@ -1,37 +1,119 @@
+        "An experienced software engineer with over seven years of experience in the industry. Currently working on a mobile app development project.",
+      skills: ["Java", "C#", "Scala", "GoLang"],
+    },
+    {
+      id: 0,
+      profilePicture:
+        "https://w7.pngwing.com/pngs/340/946/png-transparent-avatar-user-computer-icons-software-developer-avatar-child-face-heroes.png",
+      name: "Rumen Stoychev",
+      jobPosition: "Software Developer",
+      description:
+        "An experienced software engineer with over seven years of experience in the industry. Currently working on a mobile app development project.",
+      skills: ["Java", "C#", "Scala", "GoLang"],
+    },
+    {
+      id: 0,
+      profilePicture:
+        "https://w7.pngwing.com/pngs/340/946/png-transparent-avatar-user-computer-icons-software-developer-avatar-child-face-heroes.png",
+      name: "Alex Petrov",
+      jobPosition: "Software Developer",
+      description:
+        "An experienced software engineer with over seven years of experience in the industry. Currently working on a mobile app development project.",
+      skills: ["Java", "C#", "Scala", "GoLang"],
+    },
+    {
+      id: 0,
+      profilePicture:
+        "https://w7.pngwing.com/pngs/340/946/png-transparent-avatar-user-computer-icons-software-developer-avatar-child-face-heroes.png",
+      name: "Deyan Dimitrov",
+      jobPosition: "Software Developer",
+      description:
+        "An experienced software engineer with over seven years of experience in the industry. Currently working on a mobile app development project.",
+      skills: ["Java", "C#", "Scala", "GoLang"],
+    },
+  ],
+};
 
 const reducer = (state = initialState, action: AnyAction) => {
-  switch(action.type) {
+  switch (action.type) {
     case UPDATE_STATUS:
       const index = action.payload.issueId;
       const newStatus = action.payload.newStatus;
       const newState = state.issues.slice();
-      const issueToUpdate = newState.find(issue => issue.id === index);
+      const issueToUpdate = newState.find((issue) => issue.id === index);
 
       if (issueToUpdate) {
         issueToUpdate.status = newStatus;
       }
 
       return {
-        issues: newState
+        ...state,
+        issues: newState,
+      };
+    case CREATE_ISSUE:
+      const { title, description, assignee } = action.payload;
+
+      const newIssue = {
+        id: state.issues.length + 1,
+        name: title,
+        description,
+        status: "New",
+        assignee,
+      };
+
+      return {
+        ...state,
+        issues: [...state.issues, newIssue],
       };
     default:
       return state;
   }
-}
+};
 
 export const store = createStore(reducer);
 
-export type RootState = ReturnType<typeof store.getState>
-export type AppDispatch = Dispatch<UpdateStatus>
\ No newline at end of file
+export type RootState = ReturnType<typeof store.getState>;
+export type AppDispatch = Dispatch<UpdateStatus | CreateIssue>;
```

Comment:
This type will grow as the application grows. I saw in DataDash that it is done like this:
`export type AppDispatch = typeof store.dispatch;`

Reply:
done